RELEASE.md

```diff
@@ -0,0 +1,31 @@
+# Release Process
+
+You must first have [`Poetry`](https://python-poetry.org/docs/) installed.
+
+## Preparation
+
+### Version Bump and Changelog
+
+Create a pull request to prepare the release:
+
+1. Bump the `version` in `pyproject.toml`.
+1. Update `CHANGELOG.md`.
+    1. Run `git log --oneline v1.0.0..HEAD` for the commit titles, where `v.1.0.0` is the tag for the prior release.
+    1. Link to the actual PR for each commit. See the prior release notes for an example.
```

Comment:
Pants (and I think Pex) have fancy scripts to help with this. I decided to not spend the time adding that because this repo has so few commits and is very unlikely to see many updates after the initial release.